### PR TITLE
Fetchdatamodel

### PR DESCRIPTION
**Pull request by RuthDibenzi**

### CODE REVIEW

**Comment by Naveen-Jaisankar** [comment]
Could you please remove node_module from this repo...Please add that to .gitignore...

File: Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java [hunk 1]
```diff
@@ -49,7 +51,7 @@ SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
     @Bean
     CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
```
Comment (Naveen-Jaisankar) [inline Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java]: Any specific reason behind adding trailing slash?

File: DataAnalytics/model_flask_app/.idea/app.py
Comment (Naveen-Jaisankar) [inline DataAnalytics/model_flask_app/.idea/app.py:1]: We already have these files under data analytics folder....

File: FrontEnd/air-quality-app/src/App.jsx
Comment (Naveen-Jaisankar) [inline FrontEnd/air-quality-app/src/App.jsx:1]: You have remove most of the routes that are recently pushed into main branch...

File: Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java (hunk 1 above)
Comment (RuthDibenzi) [reply Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java]: ensures sub paths are covered, we don't have any at the moment but its good practice

**Review by RuthDibenzi — COMMENTED** [review]
removed duplicate files

File: DataAnalytics/model_flask_app/.idea/app.py
Comment (RuthDibenzi) [reply DataAnalytics/model_flask_app/.idea/app.py:1]: I deleted them

File: Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java (hunk 1 above)
Comment (Naveen-Jaisankar) [reply Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java]: we'll have cors issue in the production with trailing slash that's why i removed it and added a comment there

**Pull request closed without merging**